### PR TITLE
docs: polish taxi_tour_duration_prediction.md 

### PR DESCRIPTION
**Pull request by selenachenjingxin**
* **What kind of change does this PR introduce?** (Bug fix, feature, docs update, ...)

* **What is the current behavior?** (You can also link to an open issue here)

* **What is the new behavior (if this is a feature change)?**

### CODE REVIEW

**Comment by selenachenjingxin** [comment]
出租车行程预测代码全部可以跑通 与预期相符合

File: docs/zh/use_case/taxi_tour_duration_prediction.md [hunk 1]
```diff
@@ -1,81 +1,82 @@
-#  出租车行程时间预测（OpenMLDB + LightGBM）
+# 出租车行程时间预测 (OpenMLDB + LightGBM)
 
-本文我们将以[Kaggle上的出租车行车时间预测问题为例](https://www.kaggle.com/c/nyc-taxi-trip-duration/overview)，示范如何使用[OpenMLDB](https://github.com/4paradigm/OpenMLDB)和 LightGBM 联合来打造一个完整的机器学习应用。本案例基于 OpenMLDB 集群版进行教程演示。
+本文将以 [Kaggle 上的出租车行车时间预测问题](https://www.kaggle.com/c/nyc-taxi-trip-duration/overview)为例，示范如何使用 OpenMLDB 和 LightGBM 联合来打造一个完整的机器学习应用。
 
-注意，本文档使用的是预编译好的 docker 镜像。如果希望在自己编译和搭建的 OpenMLDB 环境下进行测试，需要配置使用我们[面向特征工程优化的 Spark 发行版](../tutorial/openmldbspark_distribution.md)。请参考相关[编译](../deploy/compile.md)（参考章节：“针对OpenMLDB优化的Spark发行版”）和[安装部署文档](../deploy/install_deploy.md)（参考章节：“部署TaskManager” - “2 修改配置文件conf/taskmanager.properties”）。
+注意，本文档使用的是预编译好的 Docker 镜像。如果希望在自己编译和搭建的 OpenMLDB 环境下进行测试，需要配置使用[面向特征工程优化的 Spark 发行版](https://openmldb.ai/docs/zh/main/tutorial/openmldbspark_distribution.html)。请参考[针对 OpenMLDB 优化的 Spark 发行版文档](https://openmldb.ai/docs/zh/main/deploy/compile.html#openmldb-spark)和[安装部署文档](https://openmldb.ai/docs/zh/main/deploy/install_deploy.html)。
```
Comment (lumianph) [inline docs/zh/use_case/taxi_tour_duration_prediction.md]: you should not change this to absolute path, we should use the relative path for the cross-reference

File: docs/zh/use_case/taxi_tour_duration_prediction.md [hunk 2]
```diff
@@ -1,81 +1,82 @@
...
 
 ```bash
 docker run -it 4pdosc/openmldb:0.7.1 bash
 ```
+
 该镜像预装了OpenMLDB，并预置了本案例所需要的所有脚本、三方库、开源工具以及训练数据。
 
 ```{note}
-注意，本教程以下的所有演示命令默认均在该已经启动的 docker 容器内运行。
+注意，本教程以下的 OpenMLDB 部分的演示命令默认均在启动的 Docker 容器 `openmldb` 内运行。
```
Comment (lumianph) [inline docs/zh/use_case/taxi_tour_duration_prediction.md]: 注意，本教程以下的演示命令默认均在启动的 Docker 容器内运行。

the name of a Docker container is not `openmldb`

File: docs/zh/use_case/taxi_tour_duration_prediction.md [hunk 3]
```diff
@@ -1,81 +1,82 @@
...
 
 ```bash
 /work/openmldb/bin/openmldb --zk_cluster=127.0.0.1:2181 --zk_root_path=/openmldb --role=sql_client
 ```
-```{note}
-注意，本教程大部分命令在 OpenMLDB CLI 下执行，为了跟普通 shell 环境做区分，在 OpenMLDB CLI 下执行的命令均使用特殊的提示符 `>` 。
-```
 
-### 1.4 预备知识：集群版的非阻塞任务
+### 预备知识
```
Comment (lumianph) [inline docs/zh/use_case/taxi_tour_duration_prediction.md]: prefer to keep the current title

File: docs/zh/use_case/taxi_tour_duration_prediction.md [hunk 4]
```diff
@@ -1,81 +1,82 @@
...
 
-集群版的部分命令是非阻塞任务，包括在线模式的 `LOAD DATA`，以及离线模式的 `LOAD DATA` ，`SELECT`，`SELECT INTO` 命令。提交任务以后可以使用相关的命令如 `SHOW JOBS`, `SHOW JOB` 来查看任务进度，详情参见[离线任务管理文档](../openmldb_sql/task_manage)。
+OpenMLDB 部分命令是异步的，如：在线/离线模式的 `LOAD DATA`、`SELECT`、`SELECT INTO` 命令。提交任务以后可以使用相关命令如 `SHOW JOBS`、`SHOW JOB` 来查看任务进度，详情参见[离线任务管理文档](../openmldb_sql/task_manage/SHOW_JOB.md)。
```
Comment (lumianph) [inline docs/zh/use_case/taxi_tour_duration_prediction.md]: change the url link ../openmldb_sql/task_manage/SHOW_JOB.md 
to 
../openmldb_sql/task_manage/index.rst

File: docs/zh/use_case/taxi_tour_duration_prediction.md [hunk 5]
```diff
@@ -1,81 +1,82 @@
...
 ```
 
-### 2.3 特征设计
+### 步骤 3：特征设计
 
-通常在设计特征前，用户需要根据机器学习的目标对数据进行分析，然后根据分析设计和调研特征。机器学习的数据分析和特征研究不是本文讨论的范畴，我们将不作展开。本文假定用户具备机器学习的基本理论知识，有解决机器学习问题的能力，能够理解SQL语法，并能够使用SQL语法构建特征。
+通常在设计特征前，用户需要根据机器学习的目标对数据进行分析，然后根据分析设计和调研特征。然而，机器学习的数据分析和特征研究并不是本文讨论的范畴。本文假定用户具备机器学习的基本理论知识，有解决机器学习问题的能力，能够理解 SQL 语法，并能够使用 SQL 语法构建特征。针对本案例，用户经过分析和调研设计了若干特征。
 
-针对本案例，用户经过分析和调研设计了若干特征：
+针对本案例，经过分析和调研设计了若干特征：
```
Comment (lumianph) [inline docs/zh/use_case/taxi_tour_duration_prediction.md]: you can remove this sentence

File: docs/zh/use_case/taxi_tour_duration_prediction.md [hunk 6]
```diff
@@ -1,81 +1,82 @@
...
 ```{note}
-注意，集群版 `LOAD DATA` 为非阻塞任务，可以使用命令 `SHOW JOBS` 查看任务运行状态，请等待任务运行成功（ `state` 转至 `FINISHED` 状态），再进行下一步操作 。
+`LOAD DATA` 为异步任务，请使用命令 `SHOW JOBS` 查看任务运行状态，等待任务运行成功（ `state` 转至 `FINISHED` 状态），再进行下一步操作 。
 ```
 
-### 2.3 特征设计
+### 步骤 3：特征设计
 
-通常在设计特征前，用户需要根据机器学习的目标对数据进行分析，然后根据分析设计和调研特征。机器学习的数据分析和特征研究不是本文讨论的范畴，我们将不作展开。本文假定用户具备机器学习的基本理论知识，有解决机器学习问题的能力，能够理解SQL语法，并能够使用SQL语法构建特征。
+通常在设计特征前，用户需要根据机器学习的目标对数据进行分析，然后根据分析设计和调研特征。然而，机器学习的数据分析和特征研究并不是本文讨论的范畴。本文假定用户具备机器学习的基本理论知识，有解决机器学习问题的能力，能够理解 SQL 语法，并能够使用 SQL 语法构建特征。针对本案例，用户经过分析和调研设计了若干特征。
```
Comment (lumianph) [inline docs/zh/use_case/taxi_tour_duration_prediction.md]: 针对本案例，用户经过分析和调研设计了若干特征。
=>
针对本案例，假设用户经过分析和调研设计了以下若干特征。

File: docs/zh/use_case/taxi_tour_duration_prediction.md [hunk 7]
```diff
@@ -118,74 +120,83 @@ w2 AS (PARTITION BY passenger_count ORDER BY pickup_datetime ROWS_RANGE BETWEEN
...
 
-```
-> quit
-```
+    ```
+    quit;
+    ```
 
-2. 在普通命令行下，执行 train.py(`/work/taxi-trip`目录中)，使用开源训练工具 `lightgbm` 基于上一步生成的离线特征表进行模型训练，训练结果存放在 `/tmp/model.txt`中。
+2. 在普通命令行下，执行 train.py（`/work/taxi-trip` 目录中），使用开源训练工具 `lightgbm` 基于上一步生成的离线特征表进行模型训练，训练结果存放在 `/tmp/model.txt` 中。
```
Comment (lumianph) [inline docs/zh/use_case/taxi_tour_duration_prediction.md]: `lightgbm` => LightGBM

File: docs/zh/use_case/taxi_tour_duration_prediction.md [hunk 8]
```diff
@@ -118,74 +120,83 @@ w2 AS (PARTITION BY passenger_count ORDER BY pickup_datetime ROWS_RANGE BETWEEN
...
-### 2.6 特征抽取SQL脚本上线
+    ```bash
+    python3 train.py /tmp/feature_data /tmp/model.txt
+    ```
 
-假定[2.3节中所设计的特征](#2.3-特征设计)在上一步的模型训练中产出的模型符合预期，那么下一步就是将该特征抽取SQL脚本部署到线上去，以提供在线的特征抽取。
+### 步骤 6：特征抽取 SQL 脚本上线
 
-1. 重新启动 OpenMLDB CLI，以进行 SQL 上线部署
+假定[步骤 3 所设计的特征](#步骤-3特征设计)在上一步的模型训练中产出的模型符合预期，那么下一步就是将该特征抽取 SQL 脚本部署到线上去，以提供在线特征抽取服务。
```
Comment (lumianph) [inline docs/zh/use_case/taxi_tour_duration_prediction.md:140]: double check this anchor working

**Review by lumianph — CHANGES_REQUESTED** [review]

File: docs/zh/use_case/taxi_tour_duration_prediction.md (hunk 2 above)
Comment (lumianph) [reply docs/zh/use_case/taxi_tour_duration_prediction.md]: 没修复，改成：注意，本教程以下的演示命令默认均在启动的 Docker 容器内运行。

File: docs/zh/use_case/taxi_tour_duration_prediction.md (hunk 4 above)
Comment (lumianph) [reply docs/zh/use_case/taxi_tour_duration_prediction.md]: 修改为：../openmldb_sql/task_manage/index.rst
编译以后应该是正常的那个index页面

File: docs/zh/use_case/taxi_tour_duration_prediction.md (hunk 7 above)
Comment (lumianph) [reply docs/zh/use_case/taxi_tour_duration_prediction.md]: not changed?

File: docs/zh/use_case/taxi_tour_duration_prediction.md (hunk 8 above)
Comment (lumianph) [reply docs/zh/use_case/taxi_tour_duration_prediction.md:140]: double checked?

**Review by lumianph — CHANGES_REQUESTED** [review]

**Review by lumianph — APPROVED** [review]

**Pull request merged**